### ISSUE TITLE
Unable to post update to couchdb

### ISSUE THREAD

**Issue by chrisjlebron** [benweet/stackedit#683]
I'm using a couchdb instance through couchappy to save documents, and have been getting the following error recently:

```
Error 403: Update time is in the future, please check your clock!
```

Console trace:

Call | Line
------|-------
Uncaught Error: Error 403: Update time is in the future, please check your clock! | main.js:25982
u | main.js:25982
(anonymous function) | main.js:26014
ot.Callbacks.d | main.js:1113
ot.Callbacks.p.fireWith | main.js:1156
i | main.js:2327
ot.ajaxTransport.ot.support.cors.e.crossDomain.send.n | main.js:2485

**Comment by chrisjlebron** [comment]
Also, unable to sync new documents to couchdb.

**Comment by benweet** [comment]
I think you need to synchronize your computer's clock with the server.

**Comment by chrisjlebron** [comment]
Did some troubleshooting to make sure my computer's clock is correct, and it seems to be fine (no problems with syncing to other services on stackedit, no problems with other sites). Thinking it might be an issue with couchappy. I'll update when I come get a conclusive answer from them.

**Comment by OgreSwamp** [comment]
Same for me. Laptop's date and time is fine.
I've tried to save document to public instance of coachdb in Safari and in Chrome - same issues:
`Error 403: Update time is in the future, please check your clock!`
I'm in GMT timezone, by the way.

**Comment by naufraghi** [comment]
Same problem here, both personal and work setup.

**Comment by benweet** [comment]
Fixed. I moved the database to smileupps (configuration should be automatically updated with version 4.3.11).

**Comment by VulumeCode** [comment]
New user here, using the public instance. I still have the problem. I can not set up the synchronization.

**Comment by yalp** [comment]
Same as @VulumeCode, clock is on ntp.

**Comment by rv816** [comment]
same problem.

**Comment by rv816** [comment]
#### Stackedit client code is generating incorrect timestamps

still throwing same error, at least with custom smileupps instance. perhaps problem is on smileupps side. its not a time zone thing, though -- most likely year

- Took timestamp from  couchdb
![image](https://cloud.githubusercontent.com/assets/3758548/8217701/fb778108-150c-11e5-981b-f4aefbdf473b.png)

- checked it in python
![image](https://cloud.githubusercontent.com/assets/3758548/8217661/c516ec70-150c-11e5-838b-3648a5d9b128.png)

#### tl;dr Timestamp that stackedit client is generating is incorrect.

**Comment by marcoCasamento** [comment]
Same problem here, but it seems to be related to the timezone; I'm on CEST and putting the clock one hour behind (so it's GMT +1 or "summer gmt") it works

**Comment by sidewinder42** [comment]
I have the same problem. In order to sync to the public CouchDB server I need to change my computer's clock to run an hour in the past!

I am in the UK (timezone GMT+1) similar to marcoC so it may be a DST-related problem?

**Comment by justinedelson** [comment]
Shouldn't this issue be reopened? Seeing the same thing from California.

**Comment by marcellmars** [comment]
same issue here... 4.3.11 stackedit.. 38.3.0 iceweasel on debian... also 4.3.12 google chrome on debian..

**Comment by millette** [comment]
This related issue #813 was still open last time I checked. Also getting the error, thought it was related to DST since that's when I tried it, but the problem still happens for me too.

**Comment by fgolemo** [comment]
Same issue, France (GMT+1), Chrome & Firefox, Ubuntu, time synced with NTP.

**Comment by sbrandtb** [comment]
I'm also experiencing the issue. Germany (CEST), Chrome +  Firefox, Windows 7, Time synchronized

**Comment by redpandatronicsuk** [comment]
In the document: documents/_design/validate/validate_doc_update, uncomment those lines:
